### PR TITLE
UIF-145 FY details: display funds and totals

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Update Fiscal year record view to display fund and related totals for that year
https://issues.folio.org/browse/UIF-145
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js [hunk 1]
```diff
@@ -23,32 +21,42 @@ import FiscalYearDetails from './FiscalYearDetails';
...
+              query: `budget.fiscalYearId=${fy.id}`,
+              limit: LIMIT_MAX,
+            },
+          });
+
+          return fy.id ? fyFundsPromise : [];
+        })
+        .then(response => setFunds(response))
+        .catch(() => {
+          showToast('ui-finance.fiscalYear.actions.load.error', 'error');
```
Comment (aliaksei-chumakou) [inline src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js]: it's better to reset state also, since old data will be displayed

Comment (aliaksei-chumakou) [reply src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js]: or maybe it's even better to do after `setIsLoading(true);`

Comment (AndreiShumski2) [reply src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js]: Thanx. Missed it.

File: src/components/FiscalYear/FiscalYearDetails/FiscalYearFunds/FiscalYearFunds.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import RelatedFunds from '../../../../common/RelatedFunds/RelatedFunds';
+
+const FiscalYearFunds = ({ funds, currency, fiscalYearId }) => {
+  const buildQuery = useMemo(() => {
```
Comment (NikitaSedyx) [inline src/components/FiscalYear/FiscalYearDetails/FiscalYearFunds/FiscalYearFunds.js]: are you sure you need `memo` here?

**Pull request merged**